integration-tests/pipelines/e2e-tests-pipeline.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: e2e-tests-pipeline
+spec:
+  params:
+    - description: 'Snapshot of the application'
+      name: SNAPSHOT
+      default: ''
+      type: string
+    - name: e2e_test_namespace
+      default: "build-templates-e2e"
+      description: "namespace where e2e-tests will be executed"
+  tasks:
+    - name: prepare-e2e-tests
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+      taskSpec:
+        params:
+        - name: SNAPSHOT
+        results:
+          - name: custom-source-build-bundle
+            description: "custom bundle for source build pipeline"
+        steps:
+          - name: e2e-tests
+            image: quay.io/susdas/appstudio-utils:with-make
+            env:
+            - name: SNAPSHOT
+              value: $(params.SNAPSHOT)
+            script: |
+              echo "SNAPSHOT: ${SNAPSHOT}"
+              export SOURCE_IMAGE=$(echo -n ${SNAPSHOT} | jq -r .components[0].containerImage)
+              echo "Built image: ${SOURCE_IMAGE}"
+              
+              git clone --branch setup-source-build https://github.com/tisutisu/e2e-tests.git
+              cd e2e-tests/
+              make setup-source-build | tee cmd_output
+              last_line_output=$(tail -n 1 cmd_output)
+              IFS='='
+              read -ra arr <<< "$last_line_output"
+              custom_source_build_bundle=${arr[1]}
+
+              echo -n ${custom_source_build_bundle} | tee $(results.custom-source-build-bundle.path)
+
+    - name: run-e2e-tests
+      params:
+        - name: e2e_test_namespace
+          value: "$(params.e2e_test_namespace)"
+        - name: app_suffix
+          value: "app-integration"
```

Comment:
Does the app_suffix have some special meaning? Maybe we could use a more specific name to be able to identify apps used for build-tasks-dockerfiles tests?

Reply:
app_suffix is used to identify the applications created in build-templates-e2e namespace, in build-definitions CI, we have used `ref-{{revision}}`, so here we are using `app-integration`, which is different so we can differentiate the applications created in build-temaplates-e2e namespace

Reply:
Maybe we can use `build-tasks-dockerfiles` as the suffix to make it more obvious?

Should we add `{{revision}}` to the suffix as well to make it unique for each commit? Or does it not matter?

Reply:
yes we can use `build-tasks-dockerfiles` to make it more obvious.
using `{{ revision }}` will not be resolved here, since it is a pipeline, need to identify a way to make it unique for each PR

Reply:
Ah, right :thinking:

Reply:
updated `app_suffix` to use `build-tasks-dockerfiles` for now